packages/ui-components/src/pages/PLP/PLP.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC, useState } from 'react';
+import { ProductTile } from '../../data-display/ProductTile'
```

Comment:
you can configure path in tsconfig, so we can import as @src/..... to avoid ../../../.....